src/getting_grain.md

```diff
@@ -50,6 +50,24 @@ curl -LO https://github.com/grain-lang/grain/releases/download/grain-v0.4.6/grai
 
 You'll either want to put it into your path or keep it inside your project and invoke with `.\grain-win-x64.exe`.
 
+## Alternatives
```

Comment:
I'm wondering if we should call this 
```suggestion
## Community
```

And then add a description about community run installation mechanisms and that the Grain team doesn't make sure they are up-to-date.

Reply:
How about this as a description?

> These installation methods are maintained by the community—they're not endorsed or supplied by the Grain team, but you may find them to be useful alternatives to the official distributions.

Reply:
I think, just by being on the website, they are "endorsed" in a way. Let's just say "they're not supplied by the Grain team, but"